Array Union/1_Array_Union.py

```diff
@@ -0,0 +1,38 @@
+def array_union(first,second):
+	try:
+		result = []
+		if first != []:
+			for number in first:
+				bull = contains(number,second)
+				if bull == True:
+					delete_element(first,number)
+				else:
+					number = int(number)
+					result.append(number)
+		if second != []:
+			for number in second:
+					number = int(number)
+					result.append(number)
+		return result
+	except ValueError:
+		if number in first:
+			delete_element(first,number)
+		elif number in second:
+			delete_element(second,number)
+		return []
+def delete_element(array,element):
+	array.remove(element)
+	return array
+
+def contains(number,second):
+	if number in second:
+		return True
+	else:
+		return False
+result = []
+first = (input("Enter numbers:")).split(",")
+second = (input("Enter numbers:")).split(",")
+while result == []:
+	result = array_union(first,second)
+if result != []:
+	print(sorted(result))
```

Comment:
Not distinct:

Enter numbers:1, 2, 4, 12, 2, 3, 6
Enter numbers:2, 8, 45, 12, 6, 7, 7, 7
**NOW**: [1, 2, 2, 3, 4, 6, 7, 7, 7, 8, 12, 45]
**EXPECTED**: [1, 2, 3, 4, 6, 7, 8, 12, 45]